.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules
 esm/
 lib/
+package-lock.json
```

Comment:
Not sure you want to do this?

Reply:
I'm relatively new to npm. I thought the package-lock was generated by npm when parsing package.json. I'll look into it

Reply:
No you're right. I'll add it pack in.

Reply:
OK. Thanks for the explanation. Reckon we keep their linter in play if possible. Would be good to extend this sort of pre-commit QA across the organisation more broadly.

Reply:
Agreed. I will work with the linter and ensure our repo conforms with the next pull to master.